tests/app/submitter/test_convert_payload_0_0_1.py

```diff
@@ -42,19 +52,28 @@ def test_convert_answers_to_payload_0_0_1_with_key_error(fake_questionnaire_stor
         RoutingPath(["block-1"], section_id="section-1", list_item_id=None)
     ]
     answer_object = convert_answers_to_payload_0_0_1(
-        fake_questionnaire_store.metadata,
-        fake_questionnaire_store.response_metadata,
-        fake_questionnaire_store.answer_store,
-        fake_questionnaire_store.list_store,
+        questionnaire_store.metadata,
+        questionnaire_store.response_metadata,
+        questionnaire_store.answer_store,
+        questionnaire_store.list_store,
         QuestionnaireSchema(questionnaire),
         full_routing_path,
     )
     assert answer_object["002"] == "2016-03-30"
     assert len(answer_object) == 1
 
 
-def test_answer_with_zero(fake_questionnaire_store):
-    fake_questionnaire_store.answer_store = AnswerStore([Answer("GHI", 0).to_dict()])
+@pytest.mark.parametrize(
+    "version",
+    (
+        "v1",
+        "v2",
```

Comment:
Minor, can we make sure we use the Enum everywhere.

Reply:
Updated these now 👍